pages/apis/graphql/graphql_cookbook.md

```diff
@@ -851,6 +852,32 @@ Query your organization's audit events. Audit events are only available to Enter
   }
 ```
 
+### Get timeboxed organization audit events 
```

Comment:
Since this still relates to audit events, I think it might be good to include it under the same heading. Thinking:

````
### Get organization audit events

Query your organization's audit events. Audit events are only available to Enterprise customers.

To get the first 500 audit events, use the following query:

```graphql
  query getOrganizationAuditEvents{
    organization(slug:"organization-slug"){
      auditEvents(first: 500){
        edges{
          node{
            type
            occurredAt
            actor{
              name
            }
            subject{
              name
              type
            }
          }
        }
      }
    }
  }
```

To get all audit events in a given period, use the `occurredAtFrom` and `occurredAtTo` filters like in the following query:

```graphql
  query getOrganizationAuditEvents{
    organization(slug:"organization-slug"){
      auditEvents(first: 500, occurredAtFrom: "2023-01-01T12:00:00.000", occurredAtTo: "2023-01-01T13:00:00.000"){
        edges{
          node{
            type
            occurredAt
            actor{
              name
            }
            subject{
              name
              type
            }
          }
        }
      }
    }
  }
```
````

What do you think?

Reply:
Just tweaked it to the above @mbelton-buildkite 🙂

Probably not needing that H2 anyway since its the same context, just filters